qa/workunits/fs/misc/acl.sh

```diff
@@ -47,4 +47,19 @@ expect_failure getfattr -n system.posix_acl_default .
 rmdir d1
 cd ..
 rmdir testdir
+
```

Comment:
This test will fail with kclient. I'll remove changes to this file and include it once we have the kernel change (which I'm working on).